daemon/info.go

```diff
@@ -133,9 +133,13 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 		v.CPUSet = sysInfo.Cpuset
 	}
 
-	if hostname, err := os.Hostname(); err == nil {
-		v.Name = hostname
+	hostname := "<unknown>"
```

Comment:
I'm not sure about `<unknown>` too. Maybe we should go with safer `""` value and just add log.